manifests/windows/install.pp

```diff
@@ -32,7 +37,7 @@
     content => template('puppet_agent/install_puppet.bat.erb')
   }->
   exec { 'install_puppet.bat':
-    command   => "${::system32}\\cmd.exe /c start /b ${::system32}\\cmd.exe /c \"${env_temp_variable}\\install_puppet.bat\"",
+    command   => "${_cmd_location} /c start /b ${_cmd_location} /c \"${env_temp_variable}\\install_puppet.bat\"",
```

Comment:
The call to `::system32` will resolve to `c:\windows\sysnative` and `c:\windows\sysnative\cmd.exe` will resolve just fine. However the rest of the args to the right `/c start /b $(_cmd_location)` are handed to the 64-bit cmd.exe process you just started up, one of which includes `c:\windows\sysnative\cmd.exe` (in the old code).

And then it states it can't find the file, but the error is from Cmd.exe, not from Puppet.